health_check.py

```
@@ -1,41 +1,51 @@
+"""Health check framework for Tanjun bot.
+
+Defines the base classes for all health checks:
+- HealthCheck: abstract base class
+- HealthCheckResult: result container
+- HealthStatus: status enum
+"""
+
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from dataclasses import dataclass, field
```

Flake8 (7.3.0)

[error] 'dataclasses.field' imported but unused

(F401)